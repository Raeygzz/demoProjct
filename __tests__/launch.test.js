import 'react-native';
import React from 'react';
import LaunchScreen from '../src/screens/welcome/launch';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  renderer.create(<LaunchScreen />);
});
