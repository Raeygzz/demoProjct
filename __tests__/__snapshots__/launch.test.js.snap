// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backfaceVisibility": "visible",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 6,
        "justifyContent": "center",
        "marginBottom": 0,
        "position": "relative",
        "zIndex": 9999,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": 56,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/components/assets/images/welcomeImage.png",
          }
        }
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "width": "100%",
          }
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "flex-end",
        "paddingBottom": 50,
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/components/assets/images/splash_background.png",
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "width": "100%",
        }
      }
    />
    <View>
      <View
        style={
          Object {
            "marginBottom": 26,
            "paddingHorizontal": 27,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffbcbc",
                "borderColor": "red",
                "borderWidth": 1,
                "height": 100,
                "justifyContent": "center",
                "margin": 10,
                "width": 300,
              },
              Object {
                "width": "100%",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#333333",
                "margin": 20,
              }
            }
          >
            TouchableNativeFeedback is not supported on this platform!
          </Text>
        </View>
      </View>
      <View>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#fff",
                "textAlign": "center",
              }
            }
          >
            Create Account
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
